app/src/main/java/com/example/android/architecture/blueprints/todoapp/data/CameraUtils.kt

```diff
@@ -0,0 +1,103 @@
+
+import android.content.Context
+import android.graphics.drawable.Drawable
+import android.widget.ImageView
+import com.bumptech.glide.Glide
+import com.bumptech.glide.RequestBuilder
+import com.bumptech.glide.RequestManager
+import com.bumptech.glide.load.DataSource
+import com.bumptech.glide.load.engine.DiskCacheStrategy
+import com.bumptech.glide.load.engine.GlideException
+import com.bumptech.glide.request.RequestListener
+import com.bumptech.glide.request.RequestOptions
+import com.bumptech.glide.request.target.Target
+import com.mysuperdispatch.android.extension.lazyFast
+import com.mysuperdispatch.android.utils.Utils
+import java.io.File
+
+class ImageLoader {
+
+  class Builder constructor(context: Context) {
+    var requestManager: RequestManager = Glide.with(context)
```

Comment:
it's necessary to move RequestManager toImageLoader constructor and then pass to Builder constructor() as argument